lib/healthlocker/slam/carer_connection.ex

```diff
@@ -5,18 +5,18 @@ defmodule Healthlocker.Slam.CarerConnection do
   alias Healthlocker.{EPJSUser, ReadOnlyRepo}
 
   embedded_schema do
-    field :first_name
-    field :last_name
+    field :forename
```

Comment:
can't say I'm a "_fan_" of `forename` over `first_name` but if that's what the PO wants... 👍